public/css/style.css

```diff
@@ -1,270 +1,340 @@
 * {
-  box-sizing: border-box;
-  font-family: "Helvetica Neue", Helvetica, sans-serif;
+    box-sizing: border-box;
+    font-family: "Helvetica Neue", Helvetica, sans-serif;
+}
+:root{
+--white: #ffffff;
+--black: #000000;
+--light-grey: #dddddd;
+--dark-grey: #333333;
+--light-blue: #337ab7;
+--red: #FF0000;
+--io-pink: #FF3366;
+--io-purple: #743DFB;
+--io-purple-dark: #0F0028;
+--io-green: #5FFFB4;
+
+--small: 768px;
 }
```

Comment:
It is good to see that you have added the variables, but then you don't use them 😔.

For example, you should replace every occurrence of `#dddddd` in this file with `var(--light-grey)`

Reply:
Done